### ISSUE TITLE
Browser Plugin Video Playback Problems (DRM?)

### ISSUE THREAD

**Issue by besweeet** [obsproject/obs-browser#355]
### Operating System Info

Windows 11

### OBS Studio Version

27.2.1

### OBS Studio Log URL

https://obsproject.com/logs/MlmYEmZnTYJmjcQd

### Expected Behavior

Upon page load, the video player would successfully begin playing video.

### Current Behavior

After several seconds of attempting to authenticate, the video player fails and displays a '6007' error.

### Steps to Reproduce

1. Will share via private messages.

### Anything else we should know?

Happy to provide more specifics via private messages, provide login information for debugging, etc.

Basically, this particular web-based streaming service doesn't like OBS' built-in browser plugin. It fails while it's authorizing playback.

Per the dev tools, the authentication request results in an HTTP 422 error. Several authentication attempts are made before is eventually gives up.

Other popular services – such as Netflix – appear to play nicely with the plugin, but not this one.

**Comment by derrod** [comment]
We do not support capturing DRM'd content, we do not ship the Widevine module. So this is entirely unsupported.

**Comment by besweeet** [comment]
That's interesting because at least several sources – even Netflix – work fine.

Since this is "open," there must be someone who can make this work for all.
